unified-runtime/scripts/ctest_parser.py

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python
```

Comment:
Is it worth bumping this to `python3`?

Reply:
Seems reasonable

Reply:
> Is it worth bumping this to `python3`?

Done.

Reply:
> Although, this is essentially unusable once the lit changes are merged. Perhaps you should remove it as part of #17998?

It should still work with some tweaks to file locations. What is this used for?

Reply:
Calculating CTS pass rates but with the `UUR_KNOW_FAILURE` changes it will always report 100% now. It's no longer necessary.

Reply:
I shall delete it then.